### DISCUSSION TITLE
What are multiple user for?

### DISCUSSION THREAD

**Discussion by jeanpaulrh** [garethgeorge/backrest#223] [Q&A] [answered]
Hi, I tried to add a second user and I noticed that apparently it has access to all resources (repositories and backups) defined by the first user. So, what's the point of having multi user auth?
Thanks

**Comment by garethgeorge** [comment, marked as answer]
Hey, you'd be right that they don't really have much use at the moment. It's possible in the future I'll add permissions on top (especially as I begin thinking about multi-host management). 

For the time being they're mostly incase you don't share a password with other people who have access to the installation.

**Reply by kreativekrise** [reply]
> It's possible in the future I'll add permissions on top

That would be awesome!
